samples/cpp/tutorial_code/core/mat_mask_operations/mat_mask_operations.cpp

```diff
@@ -89,7 +89,7 @@ void Sharpen(const Mat& myImage,Mat& Result)
         const uchar* next     = myImage.ptr<uchar>(j + 1);
 
         uchar* output = Result.ptr<uchar>(j);
-
+        output += nChannels;
         for(int i= nChannels;i < nChannels*(myImage.cols-1); ++i)
         {
             *output++ = saturate_cast<uchar>(5*current[i]
```

Comment:
It is better to replace in this way instead:

```.diff
-*output++ = ...
+output[i] = ...
```

(to avoid messing of different indexing approaches for lhs and rhs parts - bug comes from here)

Reply:
Thanks, I am a beginner of Git and rarely use C++.